doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -365,6 +365,9 @@ and ``coqtop``, unless stated otherwise:
 
 :-native-output-dir *dir*: Set the directory in which to put the aforementioned
   ``.cmxs`` for :tacn:`native_compute`. Defaults to ``.coq-native``.
+:-output-directory *dir*, -output-dir *dir*: Set the directory in which some commands,
+  such as :ref:`extraction` commands, or :cmd:`Redirect`, or :cmd:`Print Universes`,
+  may decide to output the files they create.
```

Comment:
```suggestion
:-output-directory *dir*, -output-dir *dir*: Sets the output directory for commands that
  write output to files, such as :ref:`extraction` commands, :cmd:`Redirect` and :cmd:`Print Universes`.
```

"may decide" seems too vague.  It's apt to be frustrating for the user if they use the option with a command that doesn't support it.  Can you be more precise?  Better yet, how about making sure that all relevant commands support the option?

Reply:
>  Better yet, how about making sure that all relevant commands support the option?

At the time of writing the PR, all commands that print were taken into account. I don't know how to ensure this in the future though.

> "may decide" seems too vague

The "may decide" refers to extraction which has an option to override the `-output-directory` command. Your formulation is however ok to me, so adopting.